content/posts/February-2022.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "February 2022"
+date: 2022-02-08
+weight: 64
+---
+
+## Week 5 (January 31th - February 7th)
+
+- A new option `--no-require-autosetup` for `source-git init` command has been
+  introduced. Please note that source-git repositories not using `%autosetup` may
+  not be properly initialized. ([packit#1470](https://github.com/packit/packit/pull/1470))
+- Packit-service now supports re-triggering against failed task using pull request
+  command `/packit rebuild-failed` and similar for testing farm: `/packit retest-failed`.
+  ([packit-service#1303](https://github.com/packit/packit-service/pull/1303))
+- There is a new job that you can use to automatically create a Bodhi update once
+  a Koji Build successfully finishes. ([packit-service#1332](https://github.com/packit/packit-service/pull/1332))
```

Comment:
I forgot to mention that in that PR, but I would add some `early version` disclaimer..;)

Reply:
since we don't listen to the events on prod yet, I have removed it